ccd-definition/AuthorisationCaseField.json

```diff
@@ -825,6 +832,13 @@
     "UserRole": "caseworker-publiclaw-cafcass",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "applicant",
+    "UserRole": "caseworker-publiclaw-cafcass",
+    "CRUD": "CRU"
```

Comment:
permissions for cafcass applicant need to be reverted to R

Reply:
👍